src/wasm_opt.rs

```diff
@@ -63,16 +68,20 @@ impl WasmOpt {
         }
     }
 
+    /// Set the level of size shrinking
     pub fn shrink(mut self, shrink_level: u32) -> Self {
         self.shrink_level = shrink_level;
         self
     }
 
+    /// Preserve debug info
     pub fn debug(mut self) -> Self {
         self.debug_info = true;
         self
     }
 
+    /// Execute `wasm-opt` over the give wasm binary, downloading it if necessary
+    /// into the `target` directory.
```

Comment:
You need to separate implementation details from the actual description of what this command does.

```suggestion
    /// Optimize the Wasm binary provided by path.
    ///
    /// This function will execute `wasm-opt` over the given Wasm binary, downloading it if necessary (cached into the `target` directory).
```